lib/Dredd.js

```diff
@@ -165,13 +173,19 @@ class Dredd {
     this.logger.debug('Preparing API description documents');
     this.prepareAPIdescriptions((error, apiDescriptions) => {
       if (error) { callback(error, this.stats); return; }
-      this.configuration.apiDescriptions = apiDescriptions;
 
-      // TODO https://github.com/apiaryio/dredd/issues/1191
-      const annotationsError = handleRuntimeProblems(apiDescriptions, this.logger);
-      if (annotationsError) { callback(annotationsError, this.stats); return; }
+      const loggerInfos = toLoggerInfos(apiDescriptions);
+      // FIXME: Winston 3.x supports calling .log() directly with the loggerInfo
```

Comment:
We can reference https://github.com/apiaryio/dredd/issues/1225 here to keep track of the changes.

Reply:
Added